tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -2,43 +2,79 @@
 
 from starkware.cairo.common.math import assert_le
 
+////////// no data - broken test
+
 @external
-func setup_examples() {
-    %{
-        max_examples(5)
+func setup_no_data() {
+    return ();
+}
+
+@external
+func test_no_data{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+    return ();
+}
 
+////////// only examples
```

Comment:
```suggestion
```
It is redundant with function name

Reply:
[modify cairo test style](https://github.com/software-mansion/protostar/pull/995/commits/fe79f3f1bc58c4a01034e9b71e3b04621944ba90)